Gemfile.lock

```diff
@@ -169,8 +169,8 @@ GEM
     bcrypt (3.1.16)
     bigdecimal (3.0.2)
     bindex (0.8.1)
-    bootsnap (1.9.1)
-      msgpack (~> 1.0)
+    bootsnap (1.10.1)
```

Comment:
Just because? Or for compatibility?

Reply:
Compatibility. The new Bootsnap version was updated for Ruby 3.